app/assets/stylesheets/filter.scss

```diff
@@ -0,0 +1,117 @@
+.slider-nav {
+    display: flex;
```

Comment:
you can also use bootstrap classes in the view `d-flex`.

Read more [here](https://getbootstrap.com/docs/4.0/utilities/display/#notation)